applications/prompt-proto-service-lsstcomcamsim/values-usdfprod-prompt-processing.yaml

```diff
@@ -10,30 +10,33 @@ prompt-proto-service:
   image:
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: 2.5.0
+    tag: 4.1.0
 
   instrument:
     pipelines:
       main: >-
-        (survey="ops-rehearsal-3")=[${PROMPT_PROCESSING_DIR}/pipelines/LSSTComCamSim/ApPipe-noForced.yaml,
+        (survey="BLOCK-297")=[${PROMPT_PROCESSING_DIR}/pipelines/LSSTComCamSim/ApPipe.yaml,
         ${PROMPT_PROCESSING_DIR}/pipelines/LSSTComCamSim/SingleFrame.yaml,
         ${PROMPT_PROCESSING_DIR}/pipelines/LSSTComCamSim/Isr.yaml]
         (survey="")=[]
       preprocessing: >-
-        (survey="ops-rehearsal-3")=[]
+        (survey="BLOCK-297")=[]
         (survey="")=[]
     calibRepo: s3://rubin-summit-users
 
   s3:
     imageBucket: rubin-summit
-    endpointUrl: https://s3dfrgw.slac.stanford.edu
+    endpointUrl: https://sdfembs3.sdf.slac.stanford.edu
 
   imageNotifications:
     kafkaClusterAddress: prompt-processing-2-kafka-bootstrap.kafka:9092
-    topic: rubin-prompt-processing-prod
+    topic: rubin-summit-notification
 
   apdb:
-    config: s3://rubin-summit-users/apdb_config/sql/pp_apdb_lsstcomcamsim.py
+    config: s3://rubin-summit-users/apdb_config/cassandra/pp_apdb_lsstcomcamsim_pre_or4.py
```

Comment:
Will you create a new config and keyspace without "pre" before OR4 begins?

Reply:
I believe that it has been created. And we need to remember to switch to it right before OR4 begins.

Reply:
I can confirm that `pp_apdb_lsstcomcamsim_or4` has been created, and is what we should use for OR4. I agree we should wait to switch until just before the rehearsal, in case they send more test data.